fslock.go

```diff
@@ -35,30 +58,20 @@ func Locked(confdir, lockFile string) (bool, error) {
 
 	lk, err := Lock(confdir, lockFile)
 	if err != nil {
-		// EAGAIN == someone else has the lock
-		if err == syscall.EAGAIN {
-			log.Debugf("Someone else has the lock: %s", filepath.Join(confdir, lockFile))
-			return true, nil
-		}
-		if strings.Contains(err.Error(), "resource temporarily unavailable") {
-			log.Debugf("Can't lock file: %s.\n reason: %s", filepath.Join(confdir, lockFile), err.Error())
+		errCase := err.Error()
+		if strings.Contains(errCase, "Lock is already held by us:") {
+			log.Debugf(errCase)
 			return true, nil
 		}
-
-		// we hold the lock ourselves
-		if strings.Contains(err.Error(), "already locked") {
-			log.Debugf("Lock is already held by us: %s", filepath.Join(confdir, lockFile))
+		if strings.Contains(errCase, "Someone else has the lock:") {
+			log.Debugf(errCase)
```

Comment:
This should be `log.Debug(errCase)` (otherwise, we'll parse `errCase` for string format specifiers).